src/main.c

```diff
@@ -262,77 +265,47 @@ int main(int argc, char* argv[]) {
   if (config.debug_level > 0)
     printf("NVIDIA %s, GFE %s (%s, %s)\n", server.gpuType, server.serverInfo.serverInfoGfeVersion, server.gsVersion, server.serverInfo.serverInfoAppVersion);
 
-  while (WHBProcIsRunning()) {
-    wiiu_screen_clear();
-
-    wiiu_screen_print(0, 0, "Moonlight Wii U v1.2 (Connected to %s)", config.address);
-    wiiu_screen_print(0, 1, "=======================================================");
-    wiiu_screen_print(0, 3, "Press A to stream, Press B to pair");
-
-    wiiu_screen_draw();
-
-    uint32_t btns = wiiu_input_buttons_triggered();
+  wiiu_screen_clear();
 
-    if (btns & VPAD_BUTTON_A) {
-      // stream
-      if (server.paired) {
-        enum platform system = platform_check(config.platform);
-        if (config.debug_level > 0)
-          printf("Platform %s\n", platform_name(system));
+  wiiu_screen_print(0, 0, "Moonlight Wii U v1.2 (Connected to %s)", config.address);
+  wiiu_screen_print(0, 1, "=======================================================");
 
-        if (system == 0) {
-          fprintf(stderr, "Platform '%s' not found\n", config.platform);
-          wiiu_error_exit("Platform '%s' not found\n", config.platform);
-        }
-        config.stream.supportsHevc = config.codec != CODEC_H264 && (config.codec == CODEC_HEVC || platform_supports_hevc(system));
+  wiiu_screen_draw();
 
-        stream(&server, &config, system);
-        break;
-      }
-      else {
-        printf("You must pair with the PC first\n");
-        wiiu_screen_clear();
-        wiiu_screen_print(0, 0, "You must pair with the PC first\n");
-        wiiu_screen_draw();
-        sleep(4);
-      }
-    } else if (btns & VPAD_BUTTON_B) {
-      // pair
-      char pin[5];
-      sprintf(pin, "%d%d%d%d", (int)random() % 10, (int)random() % 10, (int)random() % 10, (int)random() % 10);
-      printf("Please enter the following PIN on the target PC: %s\n", pin);
+  while (!server.paired) {
```

Comment:
Not a fan of this while loop here. If the server stops or the connection fails, the client is stuck in an endless loop and can't exit.